include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_for.h

```diff
@@ -0,0 +1,209 @@
+// -*- C++ -*-
+//===-- parallel_backend_sycl_for.h --------------------------------===//
```

Comment:
1) the line is not aligned with line 14 (80 symbols).
2) BTW, I don't remember, do we discuss that it is better way to omit filename in a file header?
because 
a) we already know filename - from the "filename.h" :-)
b) filename duplication might lead to "synchronization" issue - filename.h can be renamed, filename a a file header - no.
c) we already have many files w/o filename in a fileheader.

Reply:
I will align the line length. I've seen both ways in oneDPL with the header comment. I do not remember discussing the conventional, but I vaguely remember that it may have something to do with copyright. @timmiesmith Do we need to have the filename in the header for copyright purposes?